docs/design/storagenode-graceful-exit.md

```diff
@@ -0,0 +1,62 @@
+# Storage Node Graceful Exit Design Document
+
+
+## OVERVIEW
```

Comment:
Let's use title case in titles, to be consistent with other documents.

Reply:
@egonelbre I update the titles throughout the document